lib/MusicBrainz/Server/Entity/Util/JSON.pm

```diff
@@ -23,6 +24,17 @@ sub to_json_array {
     return undef;
 }
 
+sub to_json_hash {
```

Comment:
I was worried the name might be a bit too similar to `to_json_object` to distinguish them, but I can't think of anything better.  (If we made it more specific, I'd also want to change `to_json_array` for symmetry.)